ekip/redemption/views.py

```diff
@@ -202,15 +202,17 @@ def tables(request):
 @login_required
 def statistics(request):
 
-    educator_tickets = Educator.objects.all().aggregate(
+    # Filter out large - possibly abandoned requests.
+    educator_tickets_cleaned = Educator.objects.filter(num_students__lte=50).aggregate(
         Sum('num_students'))['num_students__sum']
 
-    if not educator_tickets:
-        educator_tickets = 0
+    if not educator_tickets_cleaned:
```

Comment:
Not sure on what's going on here. I did not make any changes to this file. I looked at [our repository](https://github.com/USGS-WiM/ekip-api/commits/spanish-translation/ekip/redemption/views.py) and it hasn't been modified. I compared it with what's in master and it's the same. I am not sure on why GitHub is showing it as though there is a difference.